app/js/components/labOrderEntry/DraftOrder/DraftList.jsx

```diff
@@ -1,20 +1,59 @@
 import React from 'react';
 import shortid from 'shortid';
 import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExclamationTriangle, faCheckSquare } from '@fortawesome/free-solid-svg-icons';
+
 
 export class DraftList extends React.Component {
+  static toggleIcon(order) {
+    let orderUrgencyIcon;
+    if (!Object.prototype.hasOwnProperty.call(order, 'urgency') || order.urgency === 'routine') {
+      orderUrgencyIcon = 'check-square';
+    } else {
+      orderUrgencyIcon = 'exclamation-triangle';
+    }
+    return orderUrgencyIcon;
+  }
+  constructor(props) {
+    super(props);
+
+    this.handleToggleDraftOrderUgency = this.handleToggleDraftOrderUgency.bind(this);
+  }
+
+
+  handleToggleDraftOrderUgency(order) {
+    const orderId = order.id;
+    let orderUrgency;
+    if (order.urgency && order.urgency === 'routine') {
+      orderUrgency = 'STAT';
+    } else {
+      orderUrgency = 'routine';
+    }
+    const draftOrder = {
+      orderId,
+      orderUrgency,
+    };
+    this.props.toggleDraftLabOrdersUgency(draftOrder);
+  }
+
   render() {
     return this.props.draftOrders.map(order => (
       <li className="draft-list small-font" key={shortid.generate()}>
         <span>{order.test}</span>
-        <span className="stay-right" />
+        <span className="stay-right">
+          <button onClick={() => this.handleToggleDraftOrderUgency(order)}>
```

Comment:
It doesn't have to be a button cos it'd look weird on the draft somehow, we should sync on this cos I have a similar dilemma with a similar ticket to use something more uniform. cool ?

Reply:
Fixed